Formula/node.rb

```diff
@@ -107,12 +89,18 @@ def post_install
       # Dirs must exist first: https://github.com/Homebrew/legacy-homebrew/issues/35969
       mkdir_p HOMEBREW_PREFIX/"share/man/#{man}"
       rm_f Dir[HOMEBREW_PREFIX/"share/man/#{man}/{npm.,npm-,npmrc.,package.json.}*"]
-      ln_sf Dir[libexec/"npm/lib/node_modules/npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
+      ln_sf Dir[libexec/"npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
     end
 
     npm_root = node_modules/"npm"
     npmrc = npm_root/"npmrc"
     npmrc.atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+
+    # This is a minor hack that let's us keep a working npm
+    # in both HOMEBREW_PREFIX & also privately in libexec.
```

Comment:
I think I might have an alternative idea to the inreplace workaround, which I will try out later today. Basically what I want to try out is doing everything exactly the same until this line but instead of doing the inreplace I want to use the version of npm we just copied over to `node_modules` to install a packed version of npm to libexec (after removing the old version from libexec).